stacks/okteto/deploy-local.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+# create namespace
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: okteto
+  labels:
+    certmanager.k8s.io/disable-validation: "true"
+    app.kubernetes.io/name: "okteto"
+EOF
+
+# set kubectl namespace
+kubectl config set-context --current --namespace=okteto
+
+# deploy okteto
+kubectl apply -f "$ROOT_DIR"/stacks/okteto/yaml/crds.yaml
+kubectl apply -f "$ROOT_DIR"/stacks/okteto/yaml/okteto.yaml
+
+# generate a random password
+PASSWORD=$(head -c 12 /dev/urandom | shasum| cut -d' ' -f1)
+kubectl label sa do-okteto-enterprise dev.okteto.com/token=$PASSWORD --overwrite
+
+# ensure services are running
+kubectl rollout status deployment/do-okteto-enterprise-api
```

Comment:
It looks like a number of deployments are created. Does it make sense to monitor the rollout status for all of them?

```
NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
do-cainjector                           1/1     1            1           22m
do-cert-manager                         1/1     1            1           22m
do-nginx-ingress-controller             2/2     2            2           22m
do-nginx-ingress-default-backend        2/2     2            2           22m
do-okteto-enterprise-api                2/2     2            2           22m
do-okteto-enterprise-frontend           2/2     2            2           22m
do-okteto-enterprise-mutation-webhook   2/2     2            2           22m
do-webhook                              1/1     1            1           22m
```